xilem/src/view/zstack.rs

```diff
@@ -0,0 +1,219 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    core::{
+        AppendVec, DynMessage, ElementSplice, Mut, SuperElement, View, ViewElement, ViewMarker,
+        ViewSequence,
+    },
+    Pod, ViewCtx,
+};
+use masonry::{
+    widget::{self, Alignment, WidgetMut},
+    Widget,
+};
+
+pub fn zstack<State, Action, Seq: ZStackSequence<State, Action>>(sequence: Seq) -> ZStack<Seq> {
+    ZStack {
+        sequence,
+        alignment: Alignment::default(),
+    }
+}
+
+pub struct ZStack<Seq> {
+    sequence: Seq,
+    alignment: Alignment,
+}
+
+impl<Seq> ZStack<Seq> {
+    #[must_use]
```

Comment:
The usual style is to place this on the struct definition.

Reply:
Fixed